doc/PROTOCOL-HTTP2.md

```diff
@@ -138,6 +138,8 @@ implementation can decode valid portions while leaving broken %-encodings as-is
 or replacing them with a replacement character (e.g., '?' or the Unicode
 replacement character).
 
+The repeated sequence of **Length-Prefixed-Message** has the same format as request.
```

Comment:
Hmm... This also applies to Custom-Metadata. Add it as well? I normally wouldn't be excited about listing all the things repeated here, but since it's just those two it seems fine.

Reply:
Good point, I actually did not realize that.

How about add a bold statement after Response section header:
**All fields shared with request are explained in the Request section.**

WDYT?

Reply:
Sounds good to me. Either way works fine.